tests/main/refresh-all-undo/task.yaml

```diff
@@ -23,6 +23,17 @@ prepare: |
     echo "And the daemon is configured to point to the fake store"
     setup_fake_store $BLOB_DIR
```

Comment:
I already tried that but I got some failures in another tests, the commit 1 contained that change. 
I can try again that so we can evaluate the errors, but as I remember as 5 tests failed because of that (not the refresh-all ones).